### PR TITLE
[feature/#186] Show app version in about screen

### LINKED ISSUE #186: show app version in the about screen
**Idea Description**
please show app version at the bottom of the about screen 

blocked by #181 

**Reference images and links**
[Figma](https://www.figma.com/file/NcSMs6dMsD88d4wOY0g3rK/Conference-App?node-id=0%3A1)
about-screenのコピー3

### PR DESCRIPTION
**Pull request by l2hyunwoo**
## Issue
- close #186

## Overview (Required)
- Show app version in about screen
- Add version name in setUpAndroid function

### CODE REVIEW

File: build-logic/src/main/kotlin/io/github/droidkaigi/confsched2022/primitive/AndroidGradleDsl.kt [hunk 1]
```diff
@@ -28,6 +28,9 @@ fun Project.setupAndroid() {
 
         defaultConfig {
             minSdk = 23
+
+            versionCode = 1
+            versionName = "1.0.0"
```
Comment (l2hyunwoo) [inline build-logic/src/main/kotlin/io/github/droidkaigi/confsched2022/primitive/AndroidGradleDsl.kt:33]: I added version name & version code in feature plugin's default config to show app version name on about screen

File: feature-about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/About.kt [hunk 2]
```diff
@@ -183,10 +201,16 @@ private fun navigateToCustomTab(url: String, context: Context) {
     }
 }
 
+private fun versionName(context: Context) = context.packageManager
+    .getPackageInfo(
+        context.packageName,
+        0
+    ).versionName
```
Comment (l2hyunwoo) [inline feature-about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/About.kt:208]: As I implemented it, I was a little suspicious that this code is right. I wanted to use ``BuildConfig.VERSION_NAME``, but I couldn't created BuildConfig even if versionCode is written on ``Project.setupAndroid()``. 

The function is working properly, but is this implementation as intended?

**Comment by takahirom** [comment]
👀

**Review by takahirom — APPROVED** [review]
Looks great!

**Comment by l2hyunwoo** [comment]
@takahirom Thanks!

**Pull request merged**